src/devices/Max31865/README.md

```diff
@@ -0,0 +1,35 @@
+﻿# MAX31865 - Resistance Temperature Detector Amplifier
+The MAX31865 device is a SPI interface Resistance Temperature Detector to Digital Converter.
+
+## Sensor Image
+![](sensor.jpg)
+
+## Known limitations
+This version only performs the one shot temperature conversion and not the continuous temperature conversion.
+
+## Usage
+```C#
```

Comment:
You need an extra enter between the title dans the code block. Better if you can add a sentence before (same behavior, an extra enter between each).
Also, please use plain csharp rather than the abbreviation. Github is quite tolerant but used with other viewers, those kind of little things just broke the preview on other platforms and IDE.